src/gpuarray/util.h

```diff
@@ -75,7 +76,7 @@ GPUARRAY_PUBLIC size_t gpuarray_get_elsize(int typecode);
  */
 GPUARRAY_PUBLIC int gpuarray_type_flags(int init, ...);
 
-GPUARRAY_PUBLIC int gpuarray_type_flagsa(unsigned int n, gpuarray_arg *arg);
+GPUARRAY_PUBLIC int gpuarray_type_flagsa(unsigned int n, gpuelemwise_arg *arg);
```

Comment:
That was a typo in the beginning.  However I am not against a name change.

I should also make that function internal.